create_patch.rb

```diff
@@ -21,7 +21,7 @@
   exit
 end
     
-if ARGV[0] == "reps"
+if ARGV[0] == "reps" or ARGV[0] == "representatives"
```

Comment:
Use `||` [instead of](https://github.com/bbatsov/ruby-style-guide#no-and-or-or) `or` here.